psake.psm1

```diff
@@ -574,11 +574,11 @@ function ConfigureBuildEnvironment {
         }
         {($_ -eq '4.5.1') -or ($_ -eq '4.5.2')} {
             $versions = @('v4.0.30319')
-            $buildToolsVersions = @('14.0', '12.0')
+            $buildToolsVersions = @('15.0', '14.0', '12.0')
         }
         {($_ -eq '4.6') -or ($_ -eq '4.6.1')} {

```

Comment:
Please expand it for Framework 4.6.2, thanks
`{($_ -eq '4.6') -or ($_ -eq '4.6.1') -or ($_ -eq '4.6.2')} {`